frontend/src/features/admin/users/ManageUsers.tsx

```diff
@@ -149,60 +131,62 @@ export const ManageUsers = () => {
   };
 
   return (
-    <div className="users-management-page">
-      <UsersFilterBar
-        value={filter}
-        organizationLookups={organizations}
-        rolesLookups={roles}
-        onChange={value => {
-          (value as any)?.organization
-            ? dispatch(
-                setUsersFilter({
-                  ...value,
-                  organization: (find(organizations, {
-                    id: +(value as any)?.organization,
-                  }) as any)?.name,
-                }),
-              )
-            : dispatch(setUsersFilter({ ...value, organization: '' }));
-          dispatch(setUsersPageIndex(0));
-        }}
-      />
-      {
-        <>
-          <Ribbon>
-            <TooltipWrapper toolTipId="export-to-excel" toolTip="Export to Excel">
-              <FileIcon>
-                <FaFileExcel data-testid="excel-icon" size={36} onClick={() => fetch('excel')} />
-              </FileIcon>
-            </TooltipWrapper>
-          </Ribbon>
-          <TableContainer fluid>
-            <Table<IUserRecord>
-              name="usersTable"
-              columns={columns}
-              pageIndex={pageIndex}
-              data={userList}
-              defaultCanSort={true}
-              pageCount={Math.ceil(pagedUsers.total / pageSize)}
-              pageSize={pageSize}
-              onRequestData={onRequestData}
-              onSortChange={(column, direction) => {
-                if (!!direction) {
-                  dispatch(setUsersPageSort({ [column]: direction }));
-                } else {
-                  dispatch(setUsersPageSort({}));
-                }
-              }}
-              sort={sort}
-              onPageSizeChange={size => dispatch(setUsersPageSize(size))}
-              loading={!(users && !users.isFetching)}
-              clickableTooltip="Click IDIR/BCeID link to view User Information page"
-            />
-          </TableContainer>
-        </>
-      }
-    </div>
+    <Styled.ListView fluid className="users-management-page">
```

Comment:
The **users-management-page** doesn't seem to be set anywhere but didn't want to delete it to avoid breakage due to the current mix of SCSS and styled components in the app. I felt this falls more into a cleanup tech-debt story. I already fixed a few stylings as I saw them but didn't wanted to expand the scope of this ticket (which is mainly about property list view not manage users list)